osf/models/metaschema.py

```diff
@@ -194,6 +194,17 @@ class Meta:
         order_with_respect_to = 'schema'
         unique_together = ('schema', 'registration_response_key')
 
+    INPUT_BLOCK_TYPES = frozenset([
+        'short-text-input',
+        'long-text-input',
+        'file-input',
+        'contributors-input',
+        'single-select-input',
+        'multi-select-input',
+        'select-input-option',
+        'select-other-option',
```

Comment:
Looking at real-world schemablocks, it seems that these aren't input blocks, so they don't get a `registration_response_key`, but they *do* inherit a `schema_block_group_key` from the question, so that's cool.